CMakeLists.txt

```diff
@@ -47,6 +47,27 @@ if(INFOWARE_USE_OPENGL AND APPLE)
 	message(WARNING "INFOWARE_USE_OPENGL specified, but compiling for Darwin, Metal will be used instead.")
 endif()
 
+if(MINGW AND CMAKE_CROSSCOMPILING)
+	set(WINE_PREFIX "$ENV{HOME}/.wine-${TOOLCHAIN_PREFIX}" CACHE PATH "Use this directory as Wine prefix")
+	set(WINE_PATH "${CMAKE_FIND_ROOT_PATH}/bin;${CMAKE_FIND_ROOT_PATH}/lib" CACHE STRING "Use this as WINEPATH")
```

Comment:
Default value for `WINE_PATH` could be adjusted to match Ubuntu. Current one is based on MSYS2, MinGW itself and e.g., Arch Linux.